### PR TITLE
Handle some namespaced scm things.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/scm.py [hunk 1]
```diff
@@ -193,6 +202,12 @@ def usernames(self, msg, **config):
             return set()
 
     def packages(self, msg, **config):
+
+        # If its not an rpms/ thing, then its not a package.
+        if 'namespace' in msg['msg']:
+            if msg['msg']['namespace'] != 'rpms':
+                return set()
+
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/scm.py]: Should we have some specific notifications/conversion for the ``*-checks`` repo? I figure some people would like to be notified of changes done in them.

**Comment by pypingou** [comment]
One question, but code wise :+1:

File: fedmsg_meta_fedora_infrastructure/scm.py (hunk 1 above)
Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/scm.py]: Hm, this is a good point.  Will do!

**Comment by pypingou** [comment]
:+1: 

We may want to open a ticket on FMN to add a filter for namespaces. Allowing notification from rpms only, or rpms and rpms-check or docker and rpms & so on.

**Pull request merged**